5-VM-Ubuntu-1-NodeTypes-Secure/azuredeploy.parameters.json

```diff
@@ -0,0 +1,66 @@
+{
+  "$schema": "http://schema.management.azure.com/schemas/2015-01-01/deploymentParameters.json",
+  "contentVersion": "1.0.0.0",
+  "parameters": {
+    "clusterLocation": {
+      "value": "westus"
+    },
+    "clusterName": {
+      "value": "GEN-UNIQUE"
+    },
+    "adminUserName": {
+      "value": "GEN-UNIQUE"
+    },
+    "adminPassword": {
+      "value": "GEN-PASSWORD"
+    },
+    "vmImagePublisher": {
```

Comment:
remove non-GEN values and rely on the defaultValues in the template

Reply:
Done